src/migrations/004.sql

```diff
@@ -0,0 +1,20 @@
+CREATE UNIQUE INDEX ON team_player (player_id);
+
+ALTER TABLE player
+  DROP COLUMN is_vouched;
+
+CREATE TABLE vouch (
+  vouched_id varchar(50) NOT NULL REFERENCES steam_user (steam_id),
+  voucher_id varchar(50) NOT NULL REFERENCES steam_user (steam_id),
+  CHECK(vouched_id != voucher_id)
+);
+
+CREATE TYPE yesnomaybe AS enum('yes', 'no', 'maybe');
```

Comment:
Enums are generally pretty frowned upon in SQL.  Reference tables are the preferred alternative.